### PR TITLE
Get help:  Make quality of life updates

### PR DESCRIPTION
**Pull request by codeincarnate**
**What is this feature?**

This PR makes a number of quality of life improvements to the **Get help** drawer. It clarifies language and improves the overall layout. It also removes the beta label as this is deprecated, and because this feature has been in the code base for two years.

**Who is this feature for?**

Users of the **Get help** drawer.

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

**Comment by codeincarnate** [comment]
This is still a work-in-progress. Accidentally forgot to hit draft 😢

**Comment by ivanortegaalba** [comment]
@codeincarnate is it ready to review? :)

File: .betterer.results [hunk 1]
```diff
@@ -3048,7 +3048,7 @@ exports[`better eslint`] = {
       [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "0"]
     ],
     "public/app/features/dashboard/components/HelpWizard/HelpWizard.tsx:5381": [
-      [0, 0, 0, "\'HorizontalGroup\' import from \'@grafana/ui\' is restricted from being used by a pattern. Use Stack component instead.", "0"],
```
Comment (ivanortegaalba) [inline .betterer.results:3051]: Can we take the opportunity to add the localization? So we can keep moving piece by piece :)

**Comment by codeincarnate** [comment]
@ivanortegaalba should be all ready to review!

**Review by oscarkilhed — APPROVED** [review]
lgtm

**Pull request merged**